pallets/task/src/lib.rs

```diff
@@ -298,7 +298,7 @@ pub mod pallet {
 			// Transfer balance amount to escrow account
 			let sub_account = Self::account_id(&task_id);
 			<T as self::Config>::Currency::transfer(&signer, &sub_account, budget,
-				ExistenceRequirement::KeepAlive)?;
+													ExistenceRequirement::KeepAlive)?;
```

Comment:
THe spacing has to be fixed here.. your code editor is spacing things automatically I guess

Reply:
Restored in next commit